### PR TITLE
[new] updated read me page

### PR DESCRIPTION
**Pull request by GustavoRSSilva**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,43 @@
 # Contracts
```
Comment (DioPires) [inline README.md:6]: #### Tests:
```diff 
+ 28 passing
```
#### Test coverage: 
80.25 %

Comment (DioPires) [inline README.md:6]: #### Tests:
```diff 
+ 28 passing
```
#### Test coverage: 
80.25 %

Comment (DioPires) [inline README.md:6]: #### Tests:
```diff 
+ 28 passing
```
#### Test coverage: 
80.25 %

**Review by tcrosado — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (DioPires) [inline README.md:6]: #### Tests:
```diff 
+ 28 passing
```
#### Test coverage: 
80.25 %

Comment (DioPires) [inline README.md:6]: #### Tests:
```diff 
+ 28 passing
```
#### Test coverage: 
80.25 %

**Review by DioPires — APPROVED** [review]

**Pull request merged**